_studio/mfx_lib/encode_hw/vp9/src/mfx_vp9_encode_hw_vaapi.cpp

```diff
@@ -1281,7 +1281,11 @@ mfxStatus VAAPIEncoder::QueryStatus(
 
     VASurfaceStatus surfSts = VASurfaceSkipped;
 
+#if VA_CHECK_VERSION(1,9,0)
```

Comment:
@aobolensk : please, submit an issue which you see with vp9e 8k. need to withhold with the merge.

Reply:
Created issue #2392